pipeline/acquisition.py

```diff
@@ -0,0 +1,158 @@
+'''
+Schema of aquisition information.
+'''
+import datajoint as dj
+from pipeline import reference, subject
```

Comment:
Within a package, refer to constituent modules with relative imports and do not use absolute import.
```suggestion
from . import reference, subject
```